doc/source/docs/user_guide/projections.rst

```diff
@@ -22,9 +22,9 @@ For reference codes of the most commonly used projections, see `spatialreference
 
 The same CRS can often be referred to in many ways. For example, one of the most
 commonly used CRS is the WGS84 latitude-longitude projection. This can be
-referred to using the authority code ``"EPSG:4326"``.
+referred to using the authority code "EPSG:4326".
```

Comment:
I would keep it formatted as a code, because it refers to the string passed to the method.